source/rules/no-nested-subscribe.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { Rule } from "eslint";
+import * as es from "estree";
+import { couldBeType } from "tsutils-etc";
+import { getTypeCheckerAndNodeMap } from "../utils";
+const esquery = require("esquery");
```

Comment:
Is it ok for you to add a dep to `esquery`?
Had to import it via `require` because `esquery` doesn't have a definition file.

Reply:
Yeah, that's fine. This is the first package that I've seen in a long, long time that doesn't have an `@types` definition.

Reply:
I've added some local `@types` for `esquery` so that it can be imported instead of required.

Reply:
fyi, I added generics to the `esquery` typings, as well as `getParent`.